frontend/src/components/Product.js

```diff
@@ -32,14 +33,19 @@ const Product = props => {
           {subcategory}
         </Tag>
       </TagContainer>
+      <text>
```

Comment:
`<text>` is an invalid HTML tag. Try using `<span>` instead. In addition, the designs show a bit of margin in between the ProductID and the component above it so also add that in.